docs/articles_en/documentation/legacy-features/transition-legacy-conversion-api/legacy-conversion-api/[legacy]-convert-models-as-python-objects.rst

```diff
@@ -127,3 +137,11 @@ Example of using the ``LayoutMap`` class to change the layout of a model input:
 
    ov_model = convert_model(model, layout=LayoutMap("NCHW", "NHWC"))
 
+Example of using the ``save_model`` method to save the converted model to OpenVINO IR:
+
+.. code-block:: py
+   :force:
+
+   import openvino as ov
+   ov.save_model(ov_model, 'model.xml')
```

Comment:
This does not exist in 2022.3. Is the document intended for 2023.* ? If not, `serialize()` should work in 2022.3 and 2023:

```python
from openvino.runtime import serialize

serialize(ov_model, "model.xml")
```

Reply:
great catch, please fix)